LICENSE-THIRD-PARTY

```diff
@@ -110,7 +110,7 @@ distributions of Cargo:
     SUCH DAMAGE.
 
     The licence and distribution terms for any publically available version or
-    derivative of this code cannot be changed.  i.e. this code cannot simply be
+    derivative of this code cannot be changed.  I.e., this code cannot simply be
```

Comment:
Let's leave license text as-is, I'd say.

Reply:
Fair.